### DISCUSSION TITLE
Github Educatoin Benefit not received

### DISCUSSION THREAD

**Discussion by prachin12** [community/community#133035] [GitHub Education]
### Hi everyone,

Hello!
i have applied for github education and it was approved however i have not received the github benefit yet
i was approved on 19/07/2024 but till now i havent received the benefits
i hope this will reach to the right person and help me out with the benefits!
Thank you!

**Comment by prachin12** [comment]
sorry for posting it here 
i saw previous posts too but i am just worried !

**Comment by SamuelOUS** [comment]
Hi prachin!
It usually takes 2-4 days for the benefits to appear in the account, so you should wait another day. If it doesn't work, send the petition again :)
